### PR TITLE
Final tweaks for v0.0.3

### PR DESCRIPTION
**Pull request by yuga-cb**
### What changed? Why?
Final tweaks for v0.0.3:
- Make `encrypt` a named variable in save_wallet
- Add `inspect` and `to_s` methods to most classes
- Bump version

### CODE REVIEW

File: lib/coinbase/address.rb [hunk 1]
```diff
@@ -133,10 +133,16 @@ def transfer(amount, asset_id, destination)
       Coinbase::Transfer.new(transfer_model)
     end
 
-    # Returns the address as a string.
-    # @return [String] The address
+    # Returns a String representation of the Address.
+    # @return [String] a String representation of the Address
     def to_s
-      address_id
+      "Coinbase::Address{address_id: '#{address_id}', network_id: '#{network_id}', wallet_id: '#{wallet_id}'"
```
Comment (jazz-cb) [inline lib/coinbase/address.rb:139]: missing `}` at the end

File: lib/coinbase/faucet_transaction.rb [hunk 2]
```diff
@@ -17,5 +19,17 @@ def initialize(model)
     def transaction_hash
       model.transaction_hash
     end
+
+    # Returns a String representation of the FaucetTransaction.
+    # @return [String] a String representation of the FaucetTransaction
+    def to_s
+      "Coinbase::FaucetTransaction{transaction_hash: '#{transaction_hash}'}"
```
Comment (jazz-cb) [inline lib/coinbase/faucet_transaction.rb:26]: should we add the blockexplorer link to help? 
```
https://sepolia.basescan.org/tx/#{transaction_hash}
```

File: lib/coinbase/address.rb (hunk 1 above)
Comment (yuga-cb) [reply lib/coinbase/address.rb]: Good catch, done

File: lib/coinbase/faucet_transaction.rb (hunk 2 above)
Comment (yuga-cb) [reply lib/coinbase/faucet_transaction.rb]: Done

**Review by jazz-cb — APPROVED** [review]

**Pull request merged**